Packs/ServiceNow/Integrations/ServiceNowEventCollector/ServiceNowEventCollector.yml

```diff
@@ -0,0 +1,104 @@
+category: Analytics & SIEM
+commonfields:
+  id: ServiceNow Event Collector
+  version: -1
+sectionOrder:
+- Connect
+- Collect
+configuration:
+- display: ServiceNow URL, in the format https://company.service-now.com/
+  name: url
+  required: true
+  type: 0
+  section: Connect
+- display: Username
+  name: credentials
+  type: 9
+  required: true
+  section: Connect
+- display: Client ID
+  displaypassword: Client Secret
+  name: client_credentials
+  type: 9
+  required: false
+  section: Connect
+- display: ServiceNow API Version (e.g. 'v1')
+  name: api_version
+  type: 0
+  required: false
+  section: Connect
+- additionalinfo: Select this checkbox to use OAuth 2.0 authentication.
+  display: Use OAuth Login
+  name: use_oauth
+  type: 8
+  required: false
+  section: Connect
+- defaultvalue: '10000'
+  display: Maximum number of events per fetch
+  name: max_fetch
+  required: false
+  type: 0
+  section: Collect
+- defaultvalue: 1
+  display: Events Fetch Interval
+  name: eventFetchInterval
+  type: 19
+  section: Collect
+  advanced: true
+  required: false
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+  section: Connect
+  advanced: true
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+  section: Connect
+  advanced: true
+description: Use this integration to fetch audit logs from Service Now as Cortex XSIAM events.
+display: ServiceNow Event Collector
+name: ServiceNow Event Collector
+script:
+  commands:
+  - name: service-now-get-audit-logs
+    description: Returns audit logs events extracted from ServiceNow. This command is used for developing/debugging and is to be used with caution, as it can create events, leading to events duplication and exceeding the API request limitation.
```

Comment:
```suggestion
    description: Returns audit logs events extracted from ServiceNow. This command is used for developing/debugging and is to be used with caution, as it can create events, leading to event duplication and exceeding the API request limitation.
```